src/ring_theory/noetherian.lean

```diff
@@ -792,8 +792,7 @@ section mul
 variables {R : Type*} {A : Type*} [comm_semiring R] [semiring A] [algebra R A]
 variables (M N : submodule R A)
 
-theorem fg_mul (hm : M.fg) (hn : N.fg) : (M * N).fg :=
-hm.map₂ _ hn
+theorem fg_mul (hm : M.fg) (hn : N.fg) : (M * N).fg := hm.map₂ _ hn
```

Comment:
A cute way to golf further.
```suggestion
theorem fg_mul : M.fg → N.fg → (M * N).fg := fg.map₂ _
```

Reply:
This is going to conflict with #13737 anyway; maybe I'll make that change in the merge

Reply:
I think I'd prefer to leave it as is, simply because the manual names for the hypotheses in doc-gen are nicer than the generated ones.

Reply:
`fg_mul` should be `fg.mul`, otherwise the `h.mul` below doesn't work.

Reply:
```suggestion
theorem fg.mul (hm : M.fg) (hn : N.fg) : (M * N).fg := hm.map₂ _ hn
```
Whoops, bad merge